# HG changeset patch
# User David Bateman <dbateman@free.fr>
# Date 1227914817 -3600
#      Sat Nov 29 00:26:57 2008 +0100
# Node ID 32d9c0299e9921810d52365b1db488f2458cbf07
# Parent  5798aa0f902a38d4484d21707976d454a1507752
Set gnuplot two point clipping

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2008-11-28  David Bateman  <dbateman@free.fr>
+
+	* plot/__go_draw_axes__.m: Set two point clipping mode to be on.
+
 2008-11-24  Ben Abbott <bpabbott@mac.com>
 
 	* plot/legend.m: Correct ording of legend labels.
 
 2008-11-24  Ben Abbott <bpabbott@mac.com>
 
 	* plot/__go_draw_axes__.m: Correct order for rendering children.
 
diff --git a/scripts/plot/__go_draw_axes__.m b/scripts/plot/__go_draw_axes__.m
--- a/scripts/plot/__go_draw_axes__.m
+++ b/scripts/plot/__go_draw_axes__.m
@@ -258,16 +258,18 @@ function __go_draw_axes__ (h, plot_strea
 
     xautoscale = strcmpi (axis_obj.xlimmode, "auto");
     yautoscale = strcmpi (axis_obj.ylimmode, "auto");
     zautoscale = strcmpi (axis_obj.zlimmode, "auto");
     cautoscale = strcmpi (axis_obj.climmode, "auto");
     cdatadirect = false;
     truecolor = false;
 
+    fputs (plot_stream, "set clip two;\n");
+
     kids = axis_obj.children;
 
     nd = __calc_dimensions__ (axis_obj);
 
     if (nd == 3)
       fputs (plot_stream, "set parametric;\n");
       fputs (plot_stream, "set style data lines;\n");
       fputs (plot_stream, "set surface;\n");
